scripts/populate_review_step.py

```diff
@@ -109,14 +115,21 @@ def assess_sample(self, sample):
             # Skip samples that have un-reviewed run elements - could still be sequencing and change review outcome
             return artifacts_to_upload
 
+        # skip samples which have been delivered, and mark run elements from additional runs as such
+        if self.delivered(sample.name):
+            all_artifacts = [a for a in artifacts]
+            for a in all_artifacts:
+                a.udf['RE Useable Comment'] = 'AR: Delivered, no action needed'
```

Comment:
I would reproduce the useable status and useable comment that existed before here and only add the `'AR: Delivered, no action needed'` to the one with no status/comment. Also don't forget to add the useable status as no.
Last you should add the artifacts to the `artifacts_to_upload`